opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -429,16 +429,16 @@ namespace Opm {
         // we need the inj_multiplier from the previous time step
         this->initInjMult();
 
-        const auto& summaryState = simulator_.vanguard().summaryState();
+        const auto& summaryState = this->summaryState();
         if (alternative_well_rate_init_) {
             // Update the well rates of well_state_, if only single-phase rates, to
             // have proper multi-phase rates proportional to rates at bhp zero.
             // This is done only for producers, as injectors will only have a single
             // nonzero phase anyway.
             for (const auto& well : well_container_) {
-                const bool zero_target = well->stoppedOrZeroRateTarget(simulator_, this->wellState(), local_deferredLogger);
-                if (well->isProducer() && !zero_target) {
-                    well->updateWellStateRates(simulator_, this->wellState(), local_deferredLogger);
+                //const bool zero_target = well->stoppedOrZeroRateTarget(simulator_, this->wellState(), local_deferredLogger);
+                if (well->isProducer()){// && !zero_target) {
```

Comment:
Here, I think this is the main change of this PR from https://github.com/OPM/opm-simulators/pull/5990 .  Just putting note for own reference.